array/models/actions/ActionCallContact.model.bxb

```diff
@@ -0,0 +1,11 @@
+action (ActionCallContact) {
+  description (call a single contact)
+  type (Search)
+  collect {
+    input (contact) {
+      type (StructContact)
+      min (Required) max (One)
+    }
+  }
+  output (viv.core.Text)
```

Comment:
I worry about actions that output core concepts, I worry that this will be a bad example for our devs. The output should be some concept that we can train to.

Reply:
Agree and done.